home/models.py

```diff
@@ -102,6 +102,15 @@ class Answer(models.Model):
     def __str__(self):
         return self.content
 
+    @classmethod
+    def get_Answers_by_date(cls):
+        return cls.objects.order_by('-publish_date')
+
+    @classmethod
+    def test_Save_Answer(cls, answer):
```

Comment:
Is this supposed to be a test? Then it does not belong here..

I really don;t understand what kind of logic are you trying to encapsulate here